pom.xml

```diff
@@ -221,7 +225,7 @@
             <dependency>
               <groupId>com.google.googlejavaformat</groupId>
               <artifactId>google-java-format</artifactId>
-              <version>1.7</version>
+              <version>1.9</version>
```

Comment:
Please don't change this - it requires Java 8+ and will cause the Java 7 build to fail even if it's configured to skip.

Reply:
I don't recall changing this, but intellij may have.  (Argh)